src/pages/Swap/Swap.tsx

```diff
@@ -779,58 +829,82 @@ const PriceContent: FC<{ clickNo: number; routes: any[] }> = ({ clickNo, routes
   const [outAmount, setOutAmount] = useState(0)
   const [outTokenPercentage, setOutTokenPercentage] = useState(0)
   const [cheap, setCheap] = useState(true)
-  const [tokens, setTokens] = useState([])
-
-  useEffect(() => {
-    fetch(TOKEN_LIST_URL['mainnet-beta'])
-      .then((response) => response.json())
-      .then((result) => setTokens(result))
-  }, [])
 
-  useEffect(() => {
-    const route = routes?.[clickNo]
-    if (!route) return
-    const out = route.outAmount / 10 ** tokenB.decimals
-
-    const getPriceDetails = (num: number) => {
+  const getPriceDetails = useCallback(
```

Comment:
The inputs to this function have changed and could be causing errors because they are the wrong type or the value is not being set correctly

Reply:
Will do, having a look now :)

Reply:
Good spot, and done!